app/templates/thank-you.html

```diff
@@ -2,6 +2,8 @@
 
 {% block page_title %}Submission Successful{% endblock %}
 
+{% set navigation = false %}
```

Comment:
Given each of these templates have their own get endpoints, it might make sense to set the navigation flag as part of the context on the backend?